cpu/stm32/periph/rtc_all.c

```diff
@@ -209,7 +209,7 @@ static int bcd2val(uint32_t val, int shift, uint32_t mask)
     return (((tmp >> 4) * 10) + (tmp & 0x0f));
 }
 
-static inline void rtc_unlock(void)
+extern inline void rtc_unlock(void)
```

Comment:
`extern inline void` sounds like an oxymoron to me. What does it do?

Reply:
Another copy pasta, did not think it through...

Reply:
Not sure why it was inlined should I keep that and move the function to stm32 header?

Reply:
The function does quite a few things, I think there is no reason to force it to be inlined.
We should even save some ROM this way.